### PR TITLE
Interactive

### PR DESCRIPTION
**Pull request by shadabwahidullah**
I have added local storage and I have also created a module for changing the status of tasks.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
-const toDoTasks = [
-  { desc: 'Reading a book', completed: true, index: 1 },
-  { desc: 'completing task', completed: true, index: 2 },
-  { desc: 'Climbing mountain', completed: true, index: 3 },
-];
-
 const tasks = document.querySelector('.list');
 
-function createTask(e) {
+// creates a view for task t
```
Comment (carlos-ssh) [inline src/index.js:7]: - Kindly fix this error displayed in console:
Screen Shot 2021-09-29 at 18 19 59

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @shadabwahidullah , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (shadabwahidullah) [reply src/index.js:7]: On my computer, everything works perfectly. Can you tell me when those errors appear?

Comment (shadabwahidullah) [reply src/index.js:7]: Screenshot from 2021-09-30 12-09-09

**Review by SpaYco — APPROVED** [review]
Hello 👋,

_This is an additional review as a result of the second opinion request._

You're right, I'm also not getting that error on my end, it could be that the reviewer checked another commit or an extension is causing this.

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :)

Keep rocking 🚀!

**Comment by Davidosky007** [comment]
## Hi @shadabwahidullah 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

job-welldone-gif


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**